src/storage/src/hummock/store/version.rs

```diff
@@ -653,12 +655,24 @@ impl HummockVersionReader {
             .iter_fetch_meta_duration
             .with_label_values(&[table_id_label])
             .start_timer();
+        let mut local_cache_meta_block_unhit = 0;
         let mut flatten_resps = vec![None; req_count];
         for flatten_req in flatten_reqs {
             let (req_index, resp) = flatten_req.await?;
+            local_cache_meta_block_unhit += resp.2;
             flatten_resps[req_count - req_index - 1] = Some(resp);
         }
-        timer.observe_duration();
+        let fetch_meta_duration_sec = timer.stop_and_record();
+        self.state_store_metrics
+            .iter_fetch_meta_cache_unhits
+            .with_label_values(&[table_id_label])
+            .observe(local_cache_meta_block_unhit as f64);
+        if fetch_meta_duration_sec > SLOW_ITER_FETCH_META_DURATION {
```

Comment:
Shall we need the debug log to quickly locate which table (epoch...) this query appears in?

Reply:
> Shall we need the debug log to quickly locate which table (epoch...) this query appears in?

I have just added some debug log. thanks